examples/example6/readme.md

```diff
@@ -0,0 +1,21 @@
+<!---
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+# Example 6 — use dependency management, with a separate pom.xml to define dependencies
+
+This example shows how to use a pom file for dependency management and dependencies in your project to use a bom, and then register it with 
```

Comment:
unclear. Maybe something like "This example shows how to set dependency versions in the project from a BOM"